go-controller/pkg/kubevirt/types.go

```diff
@@ -4,6 +4,8 @@ const (
 	AllowPodBridgeNetworkLiveMigrationAnnotation = "kubevirt.io/allow-pod-bridge-network-live-migration"
 	SubnetSwitchNameLabel                        = "kubevirt.io/subnet-switch-name"
 	VMLabel                                      = "kubevirt.io/vm"
+	MigrationTargetStartTimestampAnnotation      = "kubevirt.io/migration-target-start-timestamp"
```

Comment:
can you add a doc text above these explaining whats their use

Reply:
It's documented at kubevirt/api https://github.com/kubevirt/api/blob/main/core/v1/types.go#L762